assets/scripts/streets/StreetName.jsx

```diff
@@ -44,10 +45,61 @@ class StreetName extends React.PureComponent {
     super(props)
 
     this.state = {
-      isHovered: false
+      isHovered: false,
+      modify: false,
```

Comment:
For clarity, let's rename `state.modify` to `state.isEditing`. Using the prefix `is` is a common way to indicate that the property expects a boolean value, and "reads" better, e.g. `if (isEditing) { /* do stuff */ }`

The word "edit" is preferred over "modify" because it will be linguistically connected to the prop `editable` and the attribute `contenteditable` (see below).